src/main/java/frc/robot/subsystems/shooter/ShooterSubsystem.java

```diff
@@ -0,0 +1,113 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.CoastOut;
+import com.ctre.phoenix6.controls.MotionMagicVelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class ShooterSubsystem extends SubsystemBase {
+    private final TalonFX leftMotor;
+    private final TalonFX rightMotor;
+
+    private final double kS = 0.0;
+    private final double kV = 0.0;
+    private final double kA = 0.0;
+
+    private final double kP = 0.0;
+    private final double kI = 0.0;
+    private final double kD = 0.0;
+
+    private final MotionMagicVelocityVoltage LEFT_MOTION_MAGIC_VELOCITY_VOLTAGE = new MotionMagicVelocityVoltage(0, 0,
+            false, 0, 0, false, false, false);
+    private final MotionMagicVelocityVoltage RIGHT_MOTION_MAGIC_VELOCITY_VOLTAGE = new MotionMagicVelocityVoltage(0, 0,
+            false, 0, 0, false, false, false);
+
+    // value is in rotations per second
+    // TODO: TUNE THIS TO A REASONABLE VALUE
+    private final double VEOLOCITY_TOLERANCE = 0;
+
+    public ShooterSubsystem() {
+        // TODO: find can ids
+        leftMotor = new TalonFX(0);
+        rightMotor = new TalonFX(0);
+
+        TalonFXConfiguration LEFT_MOTOR_TALONFX_CONFIG = new TalonFXConfiguration();
+
+        // TODO: Check On Robot
+        LEFT_MOTOR_TALONFX_CONFIG.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kS = kS;
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kV = kV;
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kA = kA;
+
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kP = kP;
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kI = kI;
+        LEFT_MOTOR_TALONFX_CONFIG.Slot0.kD = kD;
+
+        TalonFXConfiguration RIGHT_MOTOR_TALONFX_CONFIG = new TalonFXConfiguration();
+        RIGHT_MOTOR_TALONFX_CONFIG = LEFT_MOTOR_TALONFX_CONFIG;
+        // TODO: Check On Robot
+        RIGHT_MOTOR_TALONFX_CONFIG.MotorOutput.Inverted = InvertedValue.CounterClockwise_Positive;
+
+        leftMotor.getConfigurator().apply(LEFT_MOTOR_TALONFX_CONFIG);
+        rightMotor.getConfigurator().apply(LEFT_MOTOR_TALONFX_CONFIG);
```

Comment:
This should be applying the right motor's config: `RIGHT_MOTOR_TALONFX_CONFIG`

Reply:
i'm gonna combine them into just one config, because realistically we don't need two configs, and motor direction can be set without using a configurator as far as i know

Reply:
yea, just checked the implementation of `setInverted` and it copies the current configuration before applying invert, so stuff stays the same. Because of this, I see no reason to have multiple configuration objects. Combined the two.